### PR TITLE
Guest user and old search quick fix for DSS server.

### PR DESCRIPTION
**Pull request by kevinreiss**
DSS blacklight app database table hacks.

### CODE REVIEW

File: services/dss.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+Sometimes blacklight brings unwelcome guests. Guest users and searches to be
+exact. If the database becomes too full the ```searches``` and ```users```
+tables need to be purged of the anonymous users and blank saved search values
+that each unique session started in blacklight generates. If the daily delete
+guest users and delete old searches rake tasks fail for an extended period of
+time you need to manually do this the by running the following commands in the 
+DB console:
+
+````delete from searches where user_id is NULL and updated_at <=
```
Comment (carolyncole) [inline services/dss.md]: @kevinreiss  I think you meant for this to be three ticks and a new line.  If you look a the formatted view the delete gets chopped

**Comment by carolyncole** [comment]
I still do not think this looks correct. @kevinreiss What did you really mean for folks to see?
Screen Shot 2021-05-10 at 2 33 22 PM

**Comment by tpendragon** [comment]
Did you refresh @carolyncole ? I see this:

Screen Shot 2021-05-10 at 12 05 52 PM

**Comment by hackartisan** [comment]
There is a new cron that is taking care of this now, created as a resolution to https://github.com/pulibrary/DSS/issues/180. Since that's running perhaps we don't need hard way documentation. I think we should probably close this PR. @kevinreiss what do you think?

**Review by carolyncole — APPROVED** [review]

**Pull request closed without merging**